examples/nas/enas/macro.py

```diff
@@ -18,16 +18,16 @@ def __init__(self, key, num_prev_layers, in_filters, out_filters):
             PoolBranch('avg', in_filters, out_filters, 3, 1, 1),
             PoolBranch('max', in_filters, out_filters, 3, 1, 1)
         ])
-        if num_prev_layers > 0:
-            self.skipconnect = mutables.InputChoice(num_prev_layers, n_selected=None, reduction="sum")
+        if len(prev_labels) > 0:
+            self.skipconnect = mutables.InputChoice(choose_from=prev_labels, n_chosen=None, reduction="sum")
         else:
             self.skipconnect = None
         self.batch_norm = nn.BatchNorm2d(out_filters, affine=False)
 
-    def forward(self, prev_layers, prev_labels):
+    def forward(self, prev_layers):
         out = self.mutable(prev_layers[-1])
         if self.skipconnect is not None:
-            connection = self.skipconnect(prev_layers[:-1], tags=prev_labels)
+            connection = self.skipconnect(prev_layers[:-1])
```

Comment:
any idea about how to check the mapping?

Reply:
Using a dict.

Reply:
can we check the correctness of the mapping with a dict?

Reply:
I'm doing the best I can. It's recommended to use a dict to avoid messing-up issues, but the case here is really simple.
https://github.com/microsoft/nni/blob/dfd281171c771726f452b3964982963ac2191718/src/sdk/pynni/nni/nas/pytorch/mutables.py#L130